enterprise/constants.py

```diff
@@ -60,13 +60,6 @@ class CourseModes:
     UNPAID_EXECUTIVE_EDUCATION = 'unpaid-executive-education'
 
 
-BEST_MODE_ORDER = [
-    CourseModes.VERIFIED,
-    CourseModes.PROFESSIONAL,
-    CourseModes.NO_ID_PROFESSIONAL,
-    CourseModes.UNPAID_EXECUTIVE_EDUCATION,
-]
```

Comment:
[inform/context] This removed constant has the same ordering as `COURSE_MODE_SORT_ORDER`, save for `CourseModes.UNPAID_EXECUTIVE_EDUCATION`. However, I believe this is OK as course runs with a seat mode of `CourseModes.UNPAID_EXECUTIVE_EDUCATION` do not have any seats with mode of `CourseModes.AUDIT` or `CourseModes.HONOR`.